.github/workflows/ci.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Select Xcode version
       run: sudo xcode-select -switch /Applications/Xcode_11.4.app
-    - name: Build project
-      run: swift build
+    - name: Build project (Release)
+      run: swift build -c release
```

Comment:
Curious why we need a `build` job at all? Won't `test` build the entire target anyways?

Reply:
I think build release, build + test debug is pretty good coverage

Reply:
Can't argue with that.